public-keys-component.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define(['angular'], function(angular) {
+
+'use strict';
+
+function register(module) {
+  module.component('brPublicKeys', {
```

Comment:
I think it's confusing that we have `brKeys` and `brPublicKeys`. What's the functional difference between the two? When would I use one over the other? Let's get that baked into the name instead of just adding `Public` which doesn't seem to distinguish them. Another side note is that we want to stop making two components, one for a singular display and one for a plural display, and then naming them `brFoo` and `brFoos`. It's also very confusing. A better name may be `brFooList` or `brFooTable` or something along those lines. It may be good to incorporate more information into the name as well, regarding what the component can actually do.